microsoft-edge/devtools-guide-chromium/json-viewer/json-viewer.md

```diff
@@ -0,0 +1,67 @@
+---
```

Comment:
rename the /images/ dir that's a child of the /json-viewer/ dir to /json-viewer-images/
to explicitly indicate that the dir is dedicated to that .md file.

Then in this .md, update the path for the .png files that are in that dir.